src/Command/ErrorFormatter/TableErrorFormatter.php

```diff
@@ -53,7 +56,8 @@ public function formatErrors(
 				];
 			}
 
-			$style->table(['Line', $cropFilename($file)], $rows);
+			// Note: file name may be an `int` because it may be named `10`, for example
+			$style->table(['Line', $cropFilename((string) $file)], $rows);
```

Comment:
This is needed because of wonky keys. If a file is named `10`, `ksort()` will convert the key type, and that leads to a crash here.